src/components/ebay-carousel/examples/13-continuous-title/template.marko

```diff
@@ -0,0 +1,20 @@
+<style>
```

Comment:
To be less vague, I think this file should be named `13-a11y-text-continuous`, in line with the other a11y example.